templates/web/shropshire/archive_on_report.txt

```diff
@@ -0,0 +1,7 @@
+FixMyStreet is being updated in Shropshire to improve how problems get reported.
+
+As part of this process we are closing all reports made more than 3 months before the update.
```

Comment:
It says 3 months here, but just says "before the update" in the archive email, which is correct?